src/components/custom/heading.jsx

```
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import { MainButton } from '@/components/custom/main-button';
+import { PlusIcon } from 'lucide-react';
+
+export function Heading({ title, description, actionText, onActionClick }) {
+  return (
+    <div className="max-w-5xl mx-auto">
+      <div className="flex justify-between items-center">
+        <h1 className="text-3xl fw-500">{title}</h1>
+        {actionText && <MainButton onClick={onActionClick} icon={<PlusIcon />} children={actionText} />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)